SocialCareCaseViewerApi/V1/Infrastructure/CaseStatus.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
```

Comment:
I'd personally find it really useful that if I had to work on this if it was explicit if any of these values can be null or always have a value assigned. Same for the other infrastructure classes.